libs/expo/betterangels/src/lib/helpers/generatePublicNote.ts

```diff
@@ -71,6 +68,17 @@ export default function generatePublicNote(watchedValues: IWatchedValue) {
     return item.service;
   });
 
+  const updatedP =
+    requestedServicesArray.length > 0
+      ? 'P - Follow up with client regarding requested ' +
+        requestedServicesArray.slice(0, -1).join(', ').toLowerCase() +
```

Comment:
It may be a bit difficult to understand the business logic behind this generated string, with the nested ternarys and all. Maybe break this out into separate smaller functions so it's easier to follow?

Create a generic string utility to `mergeStringsWithAnd` (if more than one) could probably help